src/Symfony/Component/Debug/ErrorHandler.php

```diff
@@ -85,6 +85,7 @@ class ErrorHandler
         E_PARSE => array(null, LogLevel::CRITICAL),
         E_ERROR => array(null, LogLevel::CRITICAL),
         E_CORE_ERROR => array(null, LogLevel::CRITICAL),
+        16777217 => array(null, LogLevel::CRITICAL)
```

Comment:
IMO is a little trickly to put these values hardcoded here. HHVM doesn't provide any constant for this?
https://github.com/facebook/hhvm/blob/de1724ed1b3bbfd6c2ad8a1a3853552030de34e0/hphp/runtime/base/runtime-error.h#L62